src/Storage/Base/IRedisSubscriberClient.cs

```diff
@@ -0,0 +1,24 @@
+using System;
+using StackExchange.Redis;
+
+namespace Snd.Sdk.Storage.Base;
+
+/// <summary>
+/// Interface for Redis subscriber operations.
+/// </summary>
+public interface IRedisSubscriberClient
```

Comment:
Can we use `IPublisher` and `ISubscriber` from the `Reactive.Streams` library? https://github.com/reactive-streams/reactive-streams-dotnet 

In this case [we can easily integrate that with Akka.NET](https://getakka.net/articles/streams/integration.html#integrating-with-reactive-streams) if we need.

Reply:
I will do the Publisher/Subscriber implementation in another PR